scripts/options/tune_editor.js

```diff
@@ -9,12 +9,14 @@ var pitchNames = [];
 var flashColor = '#FFFFFF';
 var audioContext = new AudioContext;
 
-var booper = createBooper(audioContext);
-var sampler = createSampler(audioContext);
-var tunePlayer = createTunePlayer(audioContext);
+var booper     = createBooper(audioContext);  // Instrument for the town tune editor
+var sampler    = createSampler(audioContext); // Instrument for the town tune playing at the hour
+var tunePlayer = createTunePlayer(audioContext); // Responsible for playing town tunes
 var availablePitches = tunePlayer.availablePitches;
 var rest = availablePitches[0];
 
+var defaultTownTuneVolume = 0.75; // Fallback town tune volume, change this if the default town tune volume is altered.
```

Comment:
We should find a solution not to define this default value twice

Reply:
No need.
That variable is only there to make it easy to change the default town tune volume, as we have to fetch the town tune volume twice in two separate methods in tune_editor.js, and when you fetch anything from storage you need to provide a default. 

That variable simply changes what default value you provide when fetching the volume from storage in both lines where it is necessary. I put it there so that it would be easier to change the default volume if we feel the current default volume should be changed.

Looking at the two methods that fetch the volume from storage, they cannot be incorporated into each other so that only one has to fetch the volume.